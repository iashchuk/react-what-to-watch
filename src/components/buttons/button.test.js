import React from "react";
import renderer from "react-test-renderer";
import Button from "./button";

describe(`Button`, () => {
  it(`renders correctly`, () => {
    const dom = <Button>Нажми меня</Button>;
    const component = renderer.create(dom);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`renders correctly with additional class`, () => {
    const dom = <Button className="section__button">Нажми меня</Button>;
    const component = renderer.create(dom);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
