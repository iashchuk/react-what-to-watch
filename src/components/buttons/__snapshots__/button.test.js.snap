// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="btn"
  type="button"
>
  Нажми меня
</button>
`;

exports[`Button renders correctly with additional class 1`] = `
<button
  className="btn section__button"
  type="button"
>
  Нажми меня
</button>
`;
